Doc/tutorial/introduction.rst

```diff
@@ -435,6 +428,30 @@ the :meth:`!list.append` *method* (we will see more about methods later)::
    >>> cubes
    [1, 8, 27, 64, 125, 216, 343]
 
+When you assign a variable to a list or any other object,
+it doesn't make a new copy; instead, it just points to *the same* object.
+So, any changes you make to the list through one variable will be seen
+through all other variables that point to it.::
+
+   >>> rgb = ["Red", "Green", "Blue"]
+   >>> rgba = rgb
+   >>> id(rgb) == id(rgba) # they reference the same object
+   True
+   >>> rgba.append("Alph") # changes rgb as well
+   >>> rgb
+   ["Red", "Green", "Blue", "Alph"]
```

Comment:
```suggestion
   >>> rgba.append("Alpha") # changes rgb as well
   >>> rgb
   ["Red", "Green", "Blue", "Alpha"]
```

Reply:
After replacement of `Alph`->`Alpha`, the following after this example doesn't make sense.

Reply:
Yes, I realized that after I made the comment.  It might be distracting to start with a typo-ed example like this.  Maybe there's a more realistic example?

Reply:
> Yes, I realized that after I made the comment. It might be distracting to start with a typo-ed example like this. Maybe there's a more realistic example?

It follows the style above with intentionally introduced typo (see `something's wrong here`), maybe I should've made a comment that will explicitly state that it's a typo as well. Usually there is no need to make a change in defined data unless it's wrong, but I will try to come up with better example.